clayui.com/gatsby/createPages.js

```diff
@@ -33,7 +33,7 @@ const createDocs = (actions, edges, mdx, blacklist = []) => {
 		.forEach(
 			({
 				node: {
-					fields: {layout, redirect, redirectFrom, sibling, slug},
+					fields: {layout, redirect, redirectFrom, siblings, slug},
 				},
 			}) => {
 				const hasSibling = blacklist.find(
```

Comment:
We should improve this here too, we don't want to create pages for the paths that are in the `siblings`, because we will get your content later in the template.

Reply:
Maybe this is related to the `siblings` errors being undefined, you can check the result of `siblings` in the call to `createPage` just to make sure that:

- `siblings` will always be an Array
- `siblings` will always be a String Array